simulator.py

```diff
@@ -365,13 +365,13 @@ def at(self, location, duration):
         location.humans.add(self)
         self.leaving_time = duration + self.env.now
         self.start_time = self.env.now
-
+        self.area = self.location.area
         # Report all the encounters
         for h in location.humans:
             if h == self or self.location.location_type == 'household':
                 continue
 
-            distance = self.rng.randint(50, 1000)
+            distance =  np.sqrt(int(self.area/len(self.location.humans))) + self.rng.randint(20, 850)
```

Comment:
Might be we can have a `MAX_DISTANCE_ENCOUNTER` and `MIN_DISTANCE_ENCOUNTER` parameters in `config.py` so that one can easily control it.

Reply:
Done